corporate_gifts/demo/website_theme_apply.xml

```diff
@@ -31,11 +31,4 @@
         <value model="ir.ui.view" eval="obj().env['website'].with_context(website_id=obj().env.ref('website.default_website').id).viewref('website.contactus').id"/>
         <value model="ir.ui.view" eval="{'arch': obj().env.ref('corporate_gifts.contactus').arch}"/>
     </function>
-    <function model="website.menu" name="write">
-        <value model="website.menu" eval="obj().search([
-            ('website_id', '=', ref('website.default_website')),
-            ('url', '=', '/contactus'),
-        ]).id"/>
-        <value eval="{'name': 'Request a Quote'}"/>
-    </function>
```

Comment:
Why do you remove this?

Reply:
Sorry I forgot to add it in data file :disappointed: